src/PublicApiAnalyzers/Core/Analyzers/PublicApiAnalyzerResources.resx

```diff
@@ -160,7 +160,7 @@
     <value>Symbol '{0}' is part of the declared API, but is either not public or could not be found</value>
   </data>
   <data name="RemoveDeletedApiDescription" xml:space="preserve">
-    <value>When removing a public type or member the corresponding entry in PublicAPI.txt should also be removed. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.</value>
+    <value>When removing a public type or member put that entry in PublicAPI.Unshipped.txt with *REMOVED* prefix. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.</value>
```

Comment:
Also it's safer to lock `*REMOVED*` from localization. Syntax for locking localization:

https://github.com/dotnet/roslyn-analyzers/blob/affa88c580a9cffc5b016ad892c5bcd3971e8456/src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx#L353

```suggestion
    <value>When removing a public type or member, put that entry in PublicAPI.Unshipped.txt with '*REMOVED*' prefix. This draws attention to API changes in the code reviews and source control history, and helps prevent breaking changes.</value>
```